proto/mixer.proto

```diff
@@ -237,6 +237,37 @@ message GetStatsResponse {
   string payload = 2;
 }
 
+// GetStat is a newer version of GetStats API, that use protobuf type as return
+// field instead of a "payload" of json string.
```

Comment:
nit: Its good to make the comment future proof. When we get rid of GetStats API, this will seem obsolete to future readers :)

Reply:
Done